### ISSUE TITLE
MC_returnurl parameter on PurchaseRequest

### ISSUE THREAD

**Issue by ayanozturk** [thephpleague/omnipay-worldpay#18]
Hi,

I noticed that it was commented on PurchaseRequest with this commit https://github.com/thephpleague/omnipay-worldpay/commit/b476ef534b38b697082aab916486f5c7b592d374 that `MC_returnurl` is deprecated and currently the purchase request is not sending any value for this key to worldpay. In my sandbox tests, worldpay tried to redirect to itself with `transStatus=Y` if `MC_returnurl` is missing. 

I would like to know if there is a configuration that I need to change on worldpay to return back to original requester after processing the payment without setting `MC_returnurl`?

**Comment by ayanozturk** [comment]
Also I couldn't find a page to document that MC_returnurl was deprecated. Do you have a URL for that?

**Comment by delatbabel** [comment]
> I would like to know if there is a configuration that I need to change on worldpay to return back to original requester after processing the payment without setting MC_returnurl?

No, I don't believe so. I think that `MC_returnurl` should be set from `returnUrl`.  You could try patching that and submit a PR.  I don't have a worldpay test account so I can't test it.

I don't believe the comment implied that `MC_returnurl` was deprecated, just that the use of `returnUrl` to set `MC_notifyurl` was deprecated, and that `MC_notifyurl` should be set from `notifyUrl` instead.
